mesh/meshdb.go

```diff
@@ -519,6 +519,7 @@ func (m *DB) writeTransactionRewards(l types.LayerID, accountBlockCount map[type
 	for account, smesherAccountEntry := range accountBlockCount {
 		for smesherString, cnt := range smesherAccountEntry {
 			smesherEntry, err := types.StringToNodeID(smesherString)
+			m.Info("writing reward to database")
```

Comment:
add some more context to this log message to make it more useful: layer, reward amount, smesher, etc.